pds_doi_core/util/cmd_parser.py

```diff
@@ -27,7 +27,7 @@ def create_cmd_parser():
         description='Reserve or draft a DOI\n'
                     ' Examples:\n '
                     ' % pds-doi-cmd draft -c img -s Qui.T.Chau@jpl.nasa.gov -i input/bundle_in_with_contributors.xml\n'
-                    ' % pds-doi-cmd reserve -c img -s Qui.T.Chau@jpl.nasa.gov -i input/DOI_Reserved_GEO_200318.xlsx\n',
```

Comment:
So yes the submitter should be in the command line arguments.

Reply:
The submitter_email is now a required command line argument.